frame/program.py

```diff
@@ -0,0 +1,56 @@
+import subprocess
+import os
+import signal
+from typing import Union
+
+default_wait_timeout = 10
+
+Command = Union[str, list[str]]
+
+class Program:
```

Comment:
Wonder if Program would be better as a context manager as well?

Reply:
Done